src/types/Trigger.ts

```diff
@@ -12,6 +12,28 @@ export interface TimestampTrigger {
    * The timestamp when the notification should first be shown, in milliseconds since 1970.
    */
   timestamp: number;
+
+  /**
+   * The frequency at which the trigger repeats.
+   * If unset, the notification will only be displayed once.
+   *
+   * For example:
+   *  if set to `Frequency.HOURLY`, the notification will repeat every hour from the timestamp specified.
+   *  if set to `Frequency.DAILY`, the notification will repeat every day from the timestamp specified.
+   *  if set to `Frequency.WEEKLY`, the notification will repeat every week from the timestamp specified.
+   */
+  repeatFrequency?: RepeatFrequency | RepeatFrequency.DAILY;
```

Comment:
```suggestion
   *  if set to `Frequency.WEEKLY`, the notification will repeat every week from the timestamp specified.
   *
   * Defaults to `RepeatFrequency.DAILY`
   */
  repeatFrequency?: RepeatFrequency | RepeatFrequency.DAILY;
```

Simply being a pattern-repetition machine with this suggestion, following the TimeUnit pattern below where default is spelled out

Reply:
Oops, updated it now. There's no default value.